src/main/kotlin/app/revanced/patches/youtube/layout/theme/fingerprints/LithoThemeFingerprint.kt

```diff
@@ -11,7 +11,7 @@ import org.jf.dexlib2.Opcode
 @Name("comment-actionbar-fingerprint")
 @ThemeCompatibility
 @Version("0.0.1")
-object CommentsFilterBarFingerprint : MethodFingerprint(
+object LithoThemeFingerprint : MethodFingerprint(
```

Comment:
What does the fingerprint resolve to? Is it specific place for setting the color of litho components or just the comments?

Reply:
> What does the fingerprint resolve to? Is it specific place for setting the color of litho components or just the comments?

Patch of comments filter bar and other litho elements recoloring is the same (to be extremely detailed: use the same register, placed at the same line, inside the same method of same class).

So I just renamed it.

Reply:
Please create as many but only necessary comments wherever possible to explain such namings and code for future reference.

Reply:
> Please create as many but only necessary comments wherever possible to explain such namings and code for future reference.

Why you asked me this now? This patch is practically the same as previous one, but just with hard-coded patch moved to integrations. 🤷‍♂️

Reply:
Because the PR generalizes the fingerprint. Before it's purpose was clear.

Reply:
> Because the PR generalizes the fingerprint. Before it's purpose was clear.

Oh...ok, sorry. Haven't saw the error.

...wait a bit.